types/Circus.js

```diff
@@ -32,6 +32,9 @@ export type Hook = {
 export type EventHandler = (event: Event, state: State) => void;
 
 export type Event =
+  | {|
```

Comment:
later if we want to keep `jest-circus` decoupled we should probably introduce `config` or `settings` property on the state and mutate them through a single action.
At the same time i'm a little worried about introducing one more config :D